src/index.ts

```diff
@@ -30,6 +34,10 @@ async function run() {
     await core.group("Run command", async () => {
         await matlab.runCommand(helperScript, platform, architecture, exec.exec, startupOptions);
     });
+
+    // Cleanup post run for self hosted runners
+    await io.rmRF(workspaceDir + '/.matlab');
```

Comment:
I don't think this is needed, especially if we end up putting everything in dist. All of the actions are stored in a canonical location, and if we add our plugin code to the matlab path directly from that, we don't need to copy files around or clean up files at run time.